### DISCUSSION TITLE
[Bot tutorial for singleplayer]

### DISCUSSION THREAD

**Discussion by OnestormGX1** [FakeFishGames/Barotrauma#11777] [Ideas and suggestions]
Make a tutorial for new players to teach them to use the bots correctly
[Reasoning]: Bots, orders, and order priority are not all that intuitive sometimes, green orders force your bots in ther orders target place, even the turn the reactor one. ctrl+order to make bots ignore containers it's just not explained or evident at all (thanks barotrauma wiki)

**Comment by AshEdwardsDesign** [comment]
Agreed, I'd like to see a bot-management tutorial. I had no idea you could make them ignore containers.
